gooddata-sdk/gooddata_sdk/catalog/base.py

```diff
@@ -26,7 +27,11 @@ def from_api(cls: Type[T], entity: Dict[str, Any]) -> T:
         """
         Creates object from entity passed by client class, which represents it as dictionary.
         """
-        return structure(entity, cls)
+        convertor = cls._custom_converter()
+        if convertor is not None:
+            return convertor.structure(entity, cls)
+        else:
+            return structure(entity, cls)
 
     @classmethod
     def from_dict(cls: Type[T], data: Dict[str, Any], camel_case: bool = True) -> T:
```

Comment:
Should not be custom_converter utilized also in `from_dict` method?

Reply:
That is a fair point. Let's discuss what we expect the dictionary to be because the client class works with `_from.` Should the dictionary also work with `_from`, or do we expect it to work with `from`? IMHO it could work with both, but then there will be an issue when we want to utilize `to_dict`. What then?